src/js/utils/ServiceUtil.js

```diff
@@ -54,6 +75,12 @@ const ServiceUtil = {
     appDefinition.instances = service.getInstancesCount();
     appDefinition.cmd = service.getCommand();
 
+    appDefinition.executor = service.getExecutor();
+    appDefinition.fetch = service.getFetch();
+    appDefinition.constraints = service.getConstraints();
+    appDefinition.acceptedResourceRoles = service.getAcceptedResourceRoles();
+    appDefinition.user = service.getUser();
+
     let containerSettings = service.getContainerSettings();
     if (
```

Comment:
It is a long condition

```JS
    if (
      containerSettings && containerSettings.docker &&
      containerSettings.docker.image
    ) {
```

What we could do here is
```JS

    if (containerSettings && containerSettings.docker &&
      containerSettings.docker.image
    ) {
```
My assumption was that it is more readable as it is.